recipes/hdf5/meta.yaml

```diff
@@ -26,6 +26,22 @@ requirements:
   run:
     - zlib
 
+test:
```

Comment:
Sounds like a nice idea. Probably something nifty where we import this command from an external jinja template. At this point, just want to clear this out of `master` though.